config/samples/bundle.yaml

```diff
@@ -175,6 +164,18 @@ metadata:
     type: global
   name: global-dingtalk-receiver
   namespace: kubesphere-monitoring-system
+spec:
+  chatbot:
+    conversation:
```

Comment:
conversation  is under chatbot? chatbot and conversation should be on the same level?